.github/dependabot.yml

```diff
@@ -9,10 +9,14 @@ updates:
     allow:
       - dependency-type: "all"
     assignees:
-      - "dotnet/aspnet-build"
+      - "dougbu"
+      - "juntaoluo"
+      - "wtgodbe"
+    commit-message:
+      prefix: "[main] "
```

Comment:
Will remove the trailing space in a follow-up if this results in multiple spaces.

Reply:
Why doesn't "dotnet/aspnet-build" work?

Reply:
It's ignored, likely because PRs (and issues) can't be assigned to groups.

For example, open a random issue or PR in this repo, click on `Assignees`, and start typing a group name. Pretty quickly the message below the text box will show `Nothing to show` again -- the same message shown when the box is empty.